impl/src/bin/cargo-raze.rs

```diff
@@ -120,8 +120,12 @@ fn main() -> Result<()> {
     GenMode::Vendored => bazel_renderer.render_planned_build(&render_details, &planned_build)?,
     GenMode::Remote => {
       if !dry_run {
-        // Create the "remote" directory if it doesn't exist
-        fs::create_dir_all(render_details.path_prefix.as_path().join("remote"))?;
+        // Clean out the "remote" directory and guarantee that it exists
+        let remote_dir = render_details.path_prefix.as_path().join("remote");
+        if remote_dir.exists() {
+          fs::remove_dir_all(&remote_dir)?;
```

Comment:
I agree that this would lead to a better user experience.

However, I imagine that some users might add files that were not generated by `cargo-raze` to that directory. Would it make sense to remove only files matching `*BUILD*` pattern?

Reply:
What kind of things do you think users might put in that directory?

Reply:
I don't have good answer for that, but running `rm -rf /some/directory` is something that I try to avoid if possible, and there is a clear pattern that we can match against here.

I don't feel too strongly about this change, though, so feel free to leave it as-is.

Reply:
I leave this up to @acmcarther

Reply:
I second this. Delete only BUILD files matching a regex?.

("i was too lazy to X" and "rm -rf" is a humongous red flag, btw)

Reply:
Doing the simpler thing is not a bad thing (no matter the motivation). I'll try to get this addressed later today though 😀